docs/PokemonCreation.md

```diff
@@ -0,0 +1,72 @@
+## Pokemon Creation
+
+The two main files that handle Pokemon creation are [`Equations.ts`](../src/components/Equations.ts) and [`Utilities.ts`](../src/components/Utilities.ts).
+`Equations.ts` includes functions for choosing random wild Pokemon and for creating the actual Pokemon object.
+`Utilities.ts` facilitates the creating of a wild Pokemon by preparing parameters then calling `Equations.ts` creation of the Pokemon object.
+
+A newly created Pokemon's member types can be found in [`Battles.ts`](../src/components/Battles.ts) under the `IPokemon` interface.
+
+### Equations
+
+`Equations.ts` contains the functions necessary for choosing a random Pokemon (`chooseRandomWildPokemon`), creating a new Pokemon (`newPokemon`), choosing that new Pokemon's statistics (`newPokemonStatistics`), and choosing a new Pokemon's IVs and EVs (`newPokemonIVs` and `newPokemonEvs`).
+
+### Utilities
+
+`Utilities.ts` contains the function necessary for calling `newPokemon` in `Equations.ts` (`createPokemon`), which uses the chosen random Pokemon, from `Equations.ts` `chooseRandomWildPokemon`, to get the get the level of the chosen Pokemon and title which it then feeds to `newPokemon`.
+
+### Unit Test and Trainer Battles
+
+Pass into the `newPokemon` function from `Equations.ts` an object of type `INewPokemon`.
+* `INewPokemon` holds information on the Pokemon being created and eventually helps spit out a fully created Pokemon.
```

Comment:
Lists and layers of information use bullets. Using a single bullet is weird. Just make this a second sentence in the same paragraph.